lib/trino-filesystem-gcs/src/test/java/io/trino/filesystem/gcs/AbstractTestGcsFileSystem.java

```diff
@@ -68,6 +69,13 @@ protected void initialize(String gcpCredentialKey)
         cleanupFiles();
     }
 
+    // Note: adjust settings here if tests are flaky, ex. increase maxAttempts, initial delay
```

Comment:
I don't think we should have this. If tests are flaky, that seems to indicate the defaults are not reliable and will result in failures for real queries. The defaults should be sufficient to make the tests reliable.

The tests should be a minimum bar for reliability, unless we think there is something special about the tests that makes them pathological compared to normal usage.